components/colorslider/metadata/colorslider.yml

```diff
@@ -81,40 +88,6 @@ examples:
         <input type="range" class="spectrum-ColorSlider-slider" min="0" max="100" step="1">
```

Comment:
adding the comment here since the related lines weren't change so it's not in the diff, but should we replace the inline style physical profiteers with logical ones? 

specifically the `ColorSlider-handle` has `left: 50%;`

Reply:
Good catch, this is updated. Yeah, it has to be the logical property or else it shows up in the wrong place in RTL on the docs examples.